index.html

```diff
@@ -375,26 +341,142 @@
 
     <section>
       <h2>Core Data Model</h2>
-      <p>This section describes a data model for <a>entity
-      profiles</a> and <a>entity credentials</a>, the latter covering
+
+      <p>
+This section outlines core data model concepts used throughout this
+specification
+      </p>
+
+      <section>
+        <h3>Claims</h3>
+
+        <p>
+A <a>subject</a> is an <a>entity</a> about which <a>claims</a> may be made.
+A <a>claim</a> is statement made by an <a>entity</a> about a <a>subject</a>.
```

Comment:
Fixed in https://github.com/w3c/vc-data-model/pull/67/commits/e1f1b82dcbd0124c388f3f23cc663707059c144d